File: app/src/main/java/com/example/myapplication/ui/bmi/BmiViewModel.java
package com.example.myapplication.ui.bmi;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class BmiViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public BmiViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is BMI fragment");
    }

}

File: app/src/main/java/com/example/myapplication/ui/bmi/BmiViewModel.kt
package com.example.myapplication.ui.bmi

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class BmiViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "This is BMI fragment"
    }
}